Sources/Views/SubscriptionConfirmation.swift

```diff
@@ -147,18 +167,30 @@ updateSeats()
               ])
           ],
           [
-            "You can add additional team members at anytime from your account page."
+            """
+You must have at least two seats for your team subscription. You can add additional team members at anytime
+from your account page.
```

Comment:
I don't support an owner adding themselves back, but maybe it's not worth supporting. but open to other ways of phrasing this copy

Reply:
It'd be nice but can be done later.